src/main/java/resources/PersonResource.java

```diff
@@ -115,4 +115,20 @@ public Person updatePerson(Person person) {
         personDao.update(person);
         return person;
     }
+
+    @GET
+    @Path("/recruiter")
+    @UnitOfWork
+    public Response getRecruiter() {
```

Comment:
This method return list of recruiters, so we could rename method and path to  fetchRecruiters  and /all/recruiter like above fetchPersons ?